tests/host_tools/network.py

```diff
@@ -68,35 +69,19 @@ def __init__(self, netns, ssh_key: Path, host, user, *, on_error=None):
 
         self._on_error = on_error
 
-    @property
-    def user_host(self):
-        """remote address for in SSH format <user>@<IP>"""
-        return f"{self.user}@{self.host}"
-
-    def remote_path(self, path):
-        """Convert a path to remote"""
-        return f"{self.user_host}:{path}"
-
     def _scp(self, path1, path2, options):
         """Copy files to/from the VM using scp."""
         self._exec(["scp", *options, path1, path2], check=True)
```

Comment:
nit: Do we still need `_scp()`? If no, we can also remove the internal function `_exec()` and hardcode the logic in `run()`.

Reply:
Ah, yeah, good catch! Removed :)